src/ansys/fluent/core/services/scheme_eval.py

```diff
@@ -354,12 +342,17 @@ def string_eval(self, input: str) -> str:
         response = self.service.string_eval(request)
         return response.output
 
-    def scheme_eval(self, input: str, suppress_prompts: bool = True) -> Any:
+    @deprecated(version="0.32.dev0", reason="Use ``session.scheme``.")
```

Comment:
Should this be a release version?

Reply:
@prmukherj Please update the version.

Reply:
@hpohekar Shouldn't it be some release version? Users will not be interested in dev versions.

Reply:
Yes, we can follow this criteria. (not to use dev versions)

Reply:
Done.